### PR TITLE
Check in/75711/success e2e tests

### PR DESCRIPTION
**Pull request by wullaski**
## Summary

- Adds tests for success scenarios outlined in https://github.com/department-of-veterans-affairs/va.gov-team/issues/75711

## Related issue(s)

- [department-of-veterans-affairs/va.gov-team#75711](https://github.com/department-of-veterans-affairs/va.gov-team/issues/75711)

## Testing done

- Ran created cypress tests

## What areas of the site does it impact?

Check-in Travel claim app

## Acceptance criteria

-  [ ] the tests pass

### CODE REVIEW

File: src/applications/check-in/components/BackToHome.jsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ function BackToHome() {
   const { t } = useTranslation();
   let restartURL =
     '/health-care/appointment-pre-check-in/?id=46bebc0a-b99c-464f-a5c5-560bc9eae287';
-  if (app === 'checkIn') {
+  if (app === 'dayOf') {
```
Comment (brianseek) [inline src/applications/check-in/components/BackToHome.jsx:14]: these should use APP_NAMES. i.e. `if (app === APP_NAMES.CHECK_IN)`

File: src/applications/check-in/tests/e2e/pages/TravelPages.js [hunk 2]
```diff
@@ -24,6 +24,11 @@ class TravelPages {
       .and('include.text', title);
   };
 
+  // @TODO: replace validatePageLoaded with this function
```
Comment (brianseek) [inline src/applications/check-in/tests/e2e/pages/TravelPages.js:27]: nice

File: src/applications/check-in/travel-claim/tests/e2e/multiple-facility.already-filed.cypress.spec.js
Comment (brianseek) [inline src/applications/check-in/travel-claim/tests/e2e/multiple-facility.already-filed.cypress.spec.js:1]: empty file?

Comment (wullaski) [reply src/applications/check-in/travel-claim/tests/e2e/multiple-facility.already-filed.cypress.spec.js:1]: 🗑️

**Review by ndouglas — COMMENTED** [review]
It's beautiful, NGL

**Review by brianseek — APPROVED** [review]
:shipit: Looks great!

**Pull request merged**